egs/wsj/s5/steps/dict/learn_lexicon_bayesian.sh

```diff
@@ -28,14 +28,15 @@
 
 
 # Begin configuration section.  
-cmd=run.pl
+cmd=queue.pl
```

Comment:
I don't like this to be the default since it won't work on all systems.  I'd prefer to leave the default at run.pl and pass in the current one.